lib/Common/Memory/RecyclerWriteBarrierManager.cpp

```diff
@@ -325,6 +335,11 @@ RecyclerWriteBarrierManager::WriteBarrier(void * address, size_t bytes)
     Assert(startIndex <= endIndex);
     memset(cardTable + startIndex, WRITE_BARRIER_PAGE_BIT | DIRTYBIT, endIndex - startIndex);
     GlobalSwbVerboseTrace(_u("Writing to 0x%p (CIndex: %u-%u)\n"), address, startIndex, endIndex);
+
+#if DBG
+    Recycler::WBSetBits((char*)address, (uint)bytes/sizeof(void*));
```

Comment:
bytes is size_t

---
In reply to: [92458202](https://github.com/Microsoft/ChakraCore/pull/2191#discussion_r92458202) [](ancestors = 92458202)